src/site/stages/build/process-cms-exports/transformers/node-press_releases_listing.js

```diff
@@ -2,15 +2,34 @@ const {
   createMetaTagArray,
   getDrupalValue,
   utcToEpochTime,
-  isPublished,
 } = require('./helpers');
 
+const reverseFields = reverseFieldListing => ({
+  entities: reverseFieldListing
+    ? reverseFieldListing
+        .filter(
+          reverseField =>
+            reverseField.entityBundle === 'press_release' &&
+            reverseField.status,
+        )
+        .map(reverseField => ({
+          title: reverseField.title,
+          entityUrl: reverseField.entityUrl,
+          uid: reverseField.uid,
+          fieldFeatured: reverseField.fieldFeatured,
+          fieldDate: reverseField.fieldDate,
+          fieldDescription: reverseField.fieldDescription,
+          fieldLocationHumanreadable: reverseField.fieldLocationHumanreadable,
```

Comment:
Since I copy-pasted this from another transformer, I realize these are the wrong reverse fields. But they didn't fail the validation. Not sure why.

Reply:
Oh, I see now, the `reverseField` attribute would still be set even if it's `undefined`.